standards_lab/ui/templates/project.html

```diff
@@ -182,6 +182,14 @@ <h2 class="card-title">Test</h2>
     </div>
   </div>
 
+  <div class="card mb-3" v-if="ownThisProject">
+    <div class="card-body">
+      <h2 class="card-title">Danger Zone</h2>
+      <div class="">
+        <button class="btn btn-danger" v-on:click="deleteProject" onclick="return confirm('Are you sure you want to delete? There is no undo.');">{{deleteLabel}}</button>
```

Comment:
- I think we should have the project name in the confirm message `project.name` to be really explicit about what we're deleting. "Delete project" +project.name+"?" (This can not be undone)" . 

- Could we have the `confirm()` inside `deleteProject: function(){` , ? having two onclick handlers feels odd, I don't know quite what the behaviour would be there.. e.g. are we certain of the ordering of the click event binding.  If we want to separate out the confirm to make the deleteProject function a purer function I suggest something more like:

```html
v-on:click="function(){ confirm(.....) ? deleteProject() : false }" 
```
- Is `deleteLabel` meant to be a template variable?

Reply:
First 2 - good suggestion.
3rd - just following what seemed to be current practices for other things like saveLabel.

Reply:
Ah right, yes the difference is that `saveLabel` is updated later :  https://github.com/OpenDataServices/standards-lab/blob/main/standards_lab/ui/templates/project.html#L271

Reply:
Oh, I see. I thought it was setup for clever I18N stuff or something.